app/jobs/scheduled/drop_backup_schema.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DropBackupSchema < ::Jobs::Scheduled
+    every 1.day
+
+    def execute(_)
```

Comment:
Why not `def execute`?

Reply:
The `args` parameter isn't needed in this job, but it's required. Removing the argument would lead to a `Job exception: wrong number of arguments (given 1, expected 0)`

Reply:
Ah yes of course!